### PR TITLE
removing torch_script tests

### PR DESCRIPTION
**Pull request by jjsjann123**
1. removes test_torchscript.py
2. removes nvfuser test installation in pytorch directory

### CODE REVIEW

**Comment by jjsjann123** [comment]
not sure if I needed to update any CI scripts?! I think python tests are just string matched...

**Comment by jjsjann123** [comment]
!build

**Review by xwang233 — APPROVED** [review]

**Comment by jacobhinkle** [comment]
Is it still in manual_ci.sh?

**Comment by jjsjann123** [comment]
> Is it still in manual_ci.sh?

Good catch. I'll remove that one as well.

**Comment by jjsjann123** [comment]
!build

**Comment by jjsjann123** [comment]
!build

**Comment by jjsjann123** [comment]
failure is patched on main. I'll merge this PR so we won't see nightly pip wheel failure.

File: tools/compare_codegen.sh [hunk 1]
```diff
@@ -170,8 +169,7 @@ collect_kernels() {
       fi
     else
       if [[ -d "$binarytestdir/cuda" && -d "$pyfrontenddir/cuda" &&
-          -d "$pyopsdir/cuda" && -d "$pyschedopsdir/cuda" &&
-          -d "$torchscriptdir/cuda" ]]
+          -d "$pyopsdir/cuda" && -d "$pyschedopsdir/cuda"
```
Comment (xwang233) [inline tools/compare_codegen.sh:172]: we're missing some `]]` here

Comment (jjsjann123) [reply tools/compare_codegen.sh:172]: :scream:

**Pull request merged**